tap_mongodb/sync_strategies/common.py

```diff
@@ -74,7 +74,7 @@ def string_to_class(str_value, type_value):
         return uuid.UUID(str_value)
     if type_value == 'datetime':
         return singer.utils.strptime_with_tz(str_value)
-    if type_value == 'int':
+    if type_value in ['int', 'Int64']:
```

Comment:
Is it possible to have a value in Mongo come into this block and overflow the call to `int()`?

Reply:
I don't believe so because of python's integers having arbitrary precision, but I think it makes more sense to convert an Int64 back to the python type pymongo used to represent it -- going to make that change.